File: app/src/main/java/resource/com/br/santanderapp/service/RetrofitClient.java
package resource.com.br.santanderapp.service;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class RetrofitClient {
    private static Retrofit retrofit = null;


    public static Retrofit getClient(String baseUrl) {
        if (retrofit == null) {
            retrofit = new Retrofit.Builder()
                    .baseUrl(baseUrl)
                    .addConverterFactory(GsonConverterFactory.create())
                    .build();
        }
        return retrofit;
    }
}

File: SantanderTeste/app/src/main/java/br/com/rafael/santanderteste/datasource/RetrofitClient.kt
package br.com.rafael.santanderteste.datasource

import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory

/**
 * Classe que retorna uma instancia de RetrofitClient
 */
class RetrofitClient {
    companion object {
        fun getRetrofitClient(): Retrofit {
            var retrofit = Retrofit.Builder()
                .baseUrl("https://floating-mountain-50292.herokuapp.com/")
                .addConverterFactory(GsonConverterFactory.create())
                .build()
            return retrofit
        }
    }
}